WalletWasabi.Tests/Helpers/WabiSabiFactory.cs

```diff
@@ -50,18 +50,22 @@ public static OwnershipIdentifier GetOwnershipIdentifier(Script scriptPubKey)
 		return new OwnershipIdentifier(identificationKey, scriptPubKey);
 	}
 
-	public static Round CreateRound(WabiSabiConfig cfg)
-	{
-		Round round = new(new RoundParameters(
+	public static RoundParameters CreateRoundParameters(WabiSabiConfig cfg) =>
+		RoundParameters.Create(
 			cfg,
 			Network.Main,
-			new InsecureRandom(),
 			new FeeRate(100m),
-			new CoordinationFeeRate(0.003m, Money.Zero),
-			Money.Coins(Constants.MaximumNumberOfBitcoins)));
-		round.MaxVsizeAllocationPerAlice = 11 + 31 + MultipartyTransactionParameters.SharedOverhead;
-		return round;
-	}
+			cfg.CoordinationFeeRate, // new CoordinationFeeRate(0.003m, Money.Zero),
```

Comment:
The test should use the coordination fee rate specified by the configuration instead of one hardcoded. The change is right but the comment is just the garbage leftover.